### PR TITLE
{bio}[GCCcore/8.2.0] annovar v20191024

### PR DESCRIPTION
**Pull request by seb45tian**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by seb45tian** [comment]
Test report by @seb45tian
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
r10n47.palma.wwu - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/381730aea5c013eae97738702cf4fb6c for a full test report.

File: easybuild/easyconfigs/a/annovar/annovar-2019Oct24-GCCcore-8.2.0-Perl-5.28.1.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+checksums = ['4061af3f2cc8f3d4d8187986c3d468a4b6ae145eec1858e85c7cd1059debb8ed']
+
+dependencies = [('Perl', '5.28.1')]
+
+modextrapaths = {
+    'PATH': '',
+}
+
+sanity_check_paths = {
+    'files': ["annotate_variation.pl", "retrieve_seq_from_fasta.pl"],
```
Comment (boegel) [inline easybuild/easyconfigs/a/annovar/annovar-2019Oct24-GCCcore-8.2.0-Perl-5.28.1.eb:29]: Can we include a simple sanity check command for at least one of these scripts?
Also, do they use the right `perl` command (`/usr/bin/env perl`)?

**Review by boegel — DISMISSED** [review]

File: easybuild/easyconfigs/a/annovar/annovar-2019Oct24-GCCcore-8.2.0-Perl-5.28.1.eb (hunk 1 above)
Comment (seb45tian) [reply easybuild/easyconfigs/a/annovar/annovar-2019Oct24-GCCcore-8.2.0-Perl-5.28.1.eb:29]: I could add something like 
```
annotate_variation.pl --help 2>&1 | grep Usage
```
as a sanity check for all scripts. Would this be OK?

File: easybuild/easyconfigs/a/annovar/annovar-2019Oct24-GCCcore-8.2.0-Perl-5.28.1.eb [hunk 2]
```diff
@@ -0,0 +1,33 @@
+easyblock = 'Tarball'
+
+name = 'annovar'
+# Unconventional version is taken from the upstream site
+# https://doc-openbio.readthedocs.io/projects/annovar/en/latest/user-guide/download/
+version = '2019Oct24'
```
[the remark this answers is not in the document]
Comment (seb45tian) [reply easybuild/easyconfigs/a/annovar/annovar-2019Oct24-GCCcore-8.2.0-Perl-5.28.1.eb:6]: No problem, changed with 3f675f522d727ee003c8a78b4b275635d49bcda4

File: easybuild/easyconfigs/a/annovar/annovar-2019Oct24-GCCcore-8.2.0-Perl-5.28.1.eb (hunk 1 above)
Comment (seb45tian) [reply easybuild/easyconfigs/a/annovar/annovar-2019Oct24-GCCcore-8.2.0-Perl-5.28.1.eb:29]: Forgot to mention that all scripts use `/usr/bin/env perl` .

Comment (smoors) [reply easybuild/easyconfigs/a/annovar/annovar-2019Oct24-GCCcore-8.2.0-Perl-5.28.1.eb:29]: @seb45tian ping?
the sanity check command you suggest is ok (although grepping a line a bit further down would be slightly better).

**Comment by seb45tian** [comment]
Test report by @seb45tian
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
r10n47.palma.wwu - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/4ca2b5a7488ffafb6104f58be79e2c03 for a full test report.

File: easybuild/easyconfigs/a/annovar/annovar-2019Oct24-GCCcore-8.2.0-Perl-5.28.1.eb (hunk 1 above)
Comment (seb45tian) [reply easybuild/easyconfigs/a/annovar/annovar-2019Oct24-GCCcore-8.2.0-Perl-5.28.1.eb]: All done. Thanks for the reminder.

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Comment by smoors** [comment]
Test report by @smoors
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
node351.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/a1f640f81b4acd4e9e2f8f4df0c4aadc for a full test report.

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node351.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/17edd237d0c325180514e2e3c7b62fac for a full test report.

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @seb45tian!

**Pull request merged**